### PR TITLE
reduce locoglobals - move 0x0112C876 (currentFontSpriteBase) into Gfx.h

### PR DESCRIPTION
**Pull request by LeftofZen**
an existing method was already in Gfx.h/cpp to get it

### CODE REVIEW

File: src/OpenLoco/Graphics/Gfx.cpp [hunk 1]
```diff
@@ -56,7 +56,7 @@ namespace OpenLoco::Gfx
     static loco_global<uint32_t[147], 0x050B8C8> _paletteToG1Offset;
 
     static loco_global<uint16_t, 0x112C824> _currentFontFlags;
-    static loco_global<int16_t, 0x112C876> _currentFontSpriteBase;
+    static loco_global<int32_t, 0x0112C876> _currentFontSpriteBase;
```
Comment (LeftofZen) [inline src/OpenLoco/Graphics/Gfx.cpp:59]: in some places this was int32_t and some others was int16_t. it *looks* like int16_t? but I'm unsure, @duncanspumpkin any idea?

File: src/OpenLoco/Graphics/Gfx.h [hunk 2]
```diff
@@ -277,7 +277,10 @@ namespace OpenLoco::Gfx
 
     G1Element* getG1Element(uint32_t id);
 
-    void setCurrentFontSpriteBase(int16_t value);
```
Comment (LeftofZen) [inline src/OpenLoco/Graphics/Gfx.h:280]: setter already existed here, and looking at where it's used, it makes sense to make this the new home

File: src/OpenLoco/Graphics/Gfx.cpp (hunk 1 above)
Comment (duncanspumpkin) [reply src/OpenLoco/Graphics/Gfx.cpp:59]: it should be `uint16_t`.
`0x0112C878` is used as a local variable in some dropdown function.

File: src/OpenLoco/Windows/ToolTip.cpp [hunk 3]
```diff
@@ -66,11 +64,12 @@ namespace OpenLoco::Ui::Windows::ToolTip
     {
         StringManager::formatString(_text, stringId, &args);
 
-        gCurrentFontSpriteBase = Font::medium_bold;
+        Gfx::setCurrentFontSpriteBase(Font::medium_bold);
         int16_t strWidth = Gfx::getStringWidthNewLined(_text);
         strWidth = std::max<int16_t>(strWidth, 196);
 
-        gCurrentFontSpriteBase = Font::medium_bold;
+        // same method called 3 lines up?
```
Comment (duncanspumpkin) [inline src/OpenLoco/Windows/ToolTip.cpp:71]: Yes this is needed as the global might have been set to something else by the `getStringWidthNewLined` function.

File: src/OpenLoco/Graphics/Gfx.cpp (hunk 1 above)
Comment (LeftofZen) [reply src/OpenLoco/Graphics/Gfx.cpp:59]: `uint16_t`? the values it is always compared against/assigned to are `int16_t`: https://github.com/OpenLoco/OpenLoco/blob/b2caeb64a3f9d6226fb6427bb1a7ee723ded39b1/src/OpenLoco/Localisation/StringManager.h#L19-L28

Comment (duncanspumpkin) [reply src/OpenLoco/Graphics/Gfx.cpp]: Oh yeah forgot about the m1/2 settings. int16_t

Comment (LeftofZen) [reply src/OpenLoco/Graphics/Gfx.cpp]: cool, I guessed that would be it and updated the code already

**Review by duncanspumpkin — APPROVED** [review]

**Pull request merged**